tests/manual/Param_test_lazy_load.py

```diff
@@ -0,0 +1,245 @@
+# Simplified parametrized test case using pytest fixtures
+import pytest
+import time
+import sys
+from unittest.mock import patch, MagicMock
+import os
+import time
+
+
+import shutil
+import tempfile
+
+from numpy.testing import tempdir
```

Comment:
seems to be a wrong/unused import, also clean up unused imports in general (tip: use pycharm's "Optimize Imports")